policy/policy_manager.go

```diff
@@ -175,8 +175,11 @@ func Initialize(policyPath string, arch_synonymns config.ArchSynonyms, workloadR
 	// Setup the callback functions for the policy file watcher. The change notification is the only callback
 	// that should be invoked at this time.
 	changeNotify := func(org string, fileName string, policy *Policy) {
-		numberFiles += 1
-		pm.AddPolicy(org, policy)
+		if err := pm.AddPolicy(org, policy); err != nil {
+			glog.Errorf("Unable to add policy file %v/%v during initialization, error: %v", org, fileName, err)
+		} else {
```

Comment:
You dont need these changes any more with the new code in PolicyFileChangeWatcher

Reply:
It is do guard other error, just in case.